README.md

```diff
@@ -1,94 +1,27 @@
+[Catena-X Standards](https://catena-x.net/de/standard-library) are the foundation for certifying any software product operating in the Catena-X data space. Those standards are the binding reference to obtain a valid certificate.
 
-While every repository in the [eclipse-tractusx](https://github.com/eclipse-tractusx) GitHub organization
-has its own issue management, the [Release Planning Board](https://github.com/orgs/eclipse-tractusx/projects/26)
-is used to align the overarching Tractus-X releases.
+To make adopting the latest [Catena-X Standards](https://catena-x.net/de/standard-library) easier, reference implementations are provided through the Eclipse Tractus-X Project
 
-### How can I get involved
+## Release Cycle - major vs. minor release
 
-In case you experienced a bug, unexpected behaviour, or you want to propose enhancements to Eclipse Tractus-X,
-feel free to use one of the provided [issue templates](https://github.com/eclipse-tractusx/sig-release/issues/new/choose) and describe your request.
-Please be aware, that not every feature request can be integrated and that we also cannot treat every issue with the highest priority.
+Tractus-X operates on a quarterly release cycle. Each quarter, a new [Tractus-X release](docs/tractus-x-release.md) following [CalVer](https://calver.org) is released.
 
-Every Release planning will be kicked off by two public alignment sessions. The dates and further details will be shared via
-[tractusx-dev](https://accounts.eclipse.org/mailing-list/tractusx-dev) mailing list.
-In addition to that, you can also find public meetings and info about how to join on our
-[Open Meetings](https://eclipse-tractusx.github.io/community/open-meetings) community page. 
-Issues or bug reports, that should be discussed in these meetings, have to be opened prior to the meeting via
-our [issue templates](https://github.com/eclipse-tractusx/sig-release/issues/new/choose).
+Each Tractus-X release contains [multiple products](docs/product_release.md).
 
-### What can I expect
+Once per year, Tractus-X releases a major release, making the remaining three releases minor releases. A major release may contain critical breaking changes that have a major impact on data space participants, such as changes to enablement services. A minor release contains backward compatible functionality. In addition, patch versions provide backward compatible bug and security fixes.
 
-We really welcome every contribution. Every Bug report and feature proposal takes time to prepare,
-is valuable to our project, and we very much appreciate this input.
-We are giving our best to give a first feedback in one week.
-If we should miss that, please stick with us and just use the commenting function to remind us of the issue.
+Please check the Change Log(s) for content, known knowns, and backward compatibility since the [calendar versioning scheme](https://calver.org) shows the year and month of release.
```

Comment:
Perhaps we can link to general tractus-x release changlog: https://github.com/eclipse-tractusx/tractus-x-release/blob/main/CHANGELOG.md ?